src/Button/Button.tsx

```diff
@@ -27,14 +26,24 @@ import {
  * when hovering over the button. The position, which defaults to 'left', determines if
  * the icon precedes or proceeds the provided children.
  */
-const Button: FC<ButtonComponentProps> = ({ children, ...props }) => {
-    const { alternate, color = COLORS.PRIMARY, ...extraProps } = props as ButtonProps;
-
+const Button: FC<ButtonComponentProps> = ({
+    alternate = false,
```

Comment:
I changed this because storybook will automatically document the defaultValue field in the summary table if the destructuring and assigning default values happens here.